### ISSUE TITLE
HaGeZi DNS filters name or count not shown on Updating

### ISSUE THREAD

**Issue by King-Of-Shadows** [AdguardTeam/AdguardForAndroid#5245]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.3.1

### Environment

- OS version: Android 14
- Device: OnePlus 11 5G

### HTTPS filtering

- [X] yes, I do

### Root access

- [ ] yes, I have it

### Integration with AdGuard VPN

- [ ] yes, I do

### Routing mode

Local VPN

### Ad Blocking

AdGuard Base filter, AdGuard Mobile Ads filter

### Privacy

AdGuard Tracking Protection filter, AdGuard URL Tracking filter, Legitimate URL Shortener

### Annoyances

AdGuard Annoyances filter, Adblock Warning Removal List, I don't care about cookies

### Security

Online Malicious URL Blocklist, NoCoin Filter List

### Language-specific

AdGuard Chinese filter, AdGuard Russian filter, AdGuard Ukrainian filter, IndianList

### Which DNS server do you use?

Custom DNS

### DNS protocol

None

### Custom DNS

quic://p0.freedns.controld.com
quic://unfiltered.adguard-dns.com
h3://dns.cloudflare.com/dns-query
quic://dns.nextdns.io
https://dns.quad9.net/dns-query
h3://sky.rethinkdns.com/
h3://blitz.ahadns.com

### What Stealth Mode options do you have enabled?

Block trackers, Remove tracking parameters from URLs, Hide your search queries, Ask websits not to track you, Self-destruction of third-party cookies, Block ETag and If-None-Match headers, Block the third-party Authorization header, Disable WebRTC, Block Push API, Block Location API, Hide your Referrer from third-parties, Hide your User Agent, Hide your IP address, Remove X-client-Data header from HTTP request, Protect against DPI

### Issue Details

Steps to reproduce:
1. Click Update icon on Main screen.
2. Wait for updates to complete.

### Expected Behavior

HaGeZi DNS filters name/labels and count shown on update.

### Actual Behavior

HaGeZi DNS filters name/labels and count NOT shown on update.

### Screenshots

<details><summary>Screenshot 1</summary>

![IMG_20240221_145041](https://github.com/AdguardTeam/AdguardForAndroid/assets/98694726/2204bf24-bc11-44fb-967b-1fcd246646d0)

</details>

### Additional Information

Tried DNS filters - HaGeZi pro++, TIF, DynDNS, etc. but none of them ever show getting updated on update screen.

**Comment by ESurina** [comment]
Are you sure there have been any changes to these filters? Filters are updated only if something has changed in them.

**Comment by King-Of-Shadows** [comment]
Yes, the HaGeZi DNS filters version updated dates and rules count get updated in "DNS filters" screen.

**Comment by ESurina** [comment]
I checked with my colleagues and it turned out that we do not have the option to display user filters on the Updates screen. But we liked the idea and will add this feature in the future versions!
